client/src/test/index.ts

```diff
@@ -0,0 +1,19 @@
+import { runCLI } from 'jest-cli';
+import path = require('path');
+
+const jestTestRunnerForVSCodeE2E: ITestRunner = {
```

Comment:
E2E?

Reply:
E2E → End to End; You can check [this article](https://medium.com/@soloydenko/end-to-end-testing-vs-code-extensions-via-jest-828e5edfeb75) for more information. It's the one from the first comment.

Reply:
I don't know what are recommendations for vscode extensions' testing.
This file seems to be the main input to launch tests, but isn't it possible to directly provide the jest command as a TestRunner ?